test/views/DeclarationInformationViewSpec.scala

```diff
@@ -53,7 +68,12 @@ class DeclarationInformationViewSpec extends UnitViewSpec with Injector {
 
   private val notifications = Seq(notification, rejectedNotification)
 
-  private val view = declarationInformationPage(submission, notifications)(request, messages)
+  private val declarationInformationPageWithEad = new declaration_information(gdsMainTemplate, govukSummaryList, govukTable, eadDocumentConfigEnabled)
+
+  private val declarationInformationPageWithoutEad =
+    new declaration_information(gdsMainTemplate, govukSummaryList, govukTable, eadDocumentConfigDisabled)
+
+  private val viewWithEad = declarationInformationPageWithEad(submission, notifications)(request, messages)
```

Comment:
Please have a look at `OverridableInjector':
https://github.com/hmrc/customs-movements-frontend/blob/master/test/unit/base/OverridableInjector.scala
This way you could avoid creating instances of views components by hand and only bind `EADDocumentConfig` to specific instance.